Build/_build.csproj

```diff
@@ -2,17 +2,18 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <RootNamespace></RootNamespace>
     <NoWarn>CS0649;CS0169</NoWarn>
     <NukeRootDirectory>..\</NukeRootDirectory>
     <NukeScriptDirectory>..\</NukeScriptDirectory>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Nuke.Common" Version="6.0.1" />
-    <PackageDownload Include="GitVersion.Tool" Version="[5.8.2]" />
-    <PackageDownload Include="ReportGenerator" Version="[5.0.4]" />
+    <PackageReference Include="Nuke.Common" Version="8.1.4" />
```

Comment:
🤔 Don't you want to jump to Nuke 9 instead?

Reply:
Chose nuke 8 to align with main.